### PR TITLE
Update setup.rst

### PR DESCRIPTION
**Pull request by xlefebure**

### CODE REVIEW

**Comment by javiereguiluz** [comment]
Xavier, thanks for this contribution. Personally I'm not sure about this change because the version constraint is usually defined next to the dependency ... but let's wait for more opinions.

**Comment by xlefebure** [comment]
Hi,

I tried the command in the documentation but it wasn't work for me.
The new command worked.
Let's see if other people have this problem too.

File: setup.rst [hunk 1]
```diff
@@ -251,7 +251,7 @@ create new projects. If you use Composer, you need to tell the exact version:
 
 .. code-block:: terminal
 
-    $ composer create-project symfony/website-skeleton:^4.4 my_project_name
```
Comment (HeahDude) [inline setup.rst]: There is indeed a confusion with the `require` argument syntax, could you please keep the minor version though?

**Review by HeahDude — CHANGES_REQUESTED** [review]

**Comment by javiereguiluz** [comment]
I don't know what to do here:

I've run these two commands and both worked perfectly:

```
composer create-project symfony/website-skeleton:^4.4 my_project_name

composer create-project symfony/website-skeleton:"^4.4" my_project_name
```

I'm using macOS. I remember some issues in the past with Windows. @xlefebure maybe you were using Windows when running the command?

**Comment by xlefebure** [comment]
@javiereguiluz Yes, I was on Windows 10 at this moment.

**Comment by javiereguiluz** [comment]
Can you try if this command works for you? (note the quotes around the number version):

```bash
composer create-project symfony/website-skeleton:"^4.4" my_project_name
```

**Comment by xlefebure** [comment]
Hi,

It's working !

ᐧ

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Garanti
sans virus. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

Le jeu. 12 mars 2020 à 17:55, Javier Eguiluz <notifications@github.com> a
écrit :

**Comment by javiereguiluz** [comment]
Thanks for confirming that the quotes make this problem disappear. We did that change while merging. Thanks!

**Pull request merged**